hearts/hearts.py

```diff
@@ -140,6 +141,11 @@ def deal(self):
             start_hand.hand_sort()
             self.hands[self.players[n]] = start_hand
 
+    def set_turn_counter(self):
```

Comment:
I would call this `initialize_turn_counter` to make it clear it's for the start of the game.